examples/vector_databases/Using_vector_databases_for_embeddings_search.ipynb

```diff
@@ -0,0 +1,1419 @@
+    "    query_result = index.query(embedded_query, \n",
+    "                                      namespace=namespace, \n",
+    "                                      top_k=top_k)\n",
+    "\n",
+    "    # Print query results \n",
+    "    print(f'\\nMost similar results to {query} in \"{namespace}\" namespace:\\n')\n",
+    "    if not query_result.matches:\n",
+    "        print('no query result')\n",
+    "    \n",
+    "    matches = query_result.matches\n",
+    "    ids = [res.id for res in matches]\n",
+    "    scores = [res.score for res in matches]\n",
+    "    df = pd.DataFrame({'id':ids, \n",
+    "                       'score':scores,\n",
+    "                       'title': [titles_mapped[_id] for _id in ids],\n",
+    "                       'content': [content_mapped[_id] for _id in ids],\n",
+    "                       })\n",
+    "    \n",
+    "    counter = 0\n",
+    "    for k,v in df.iterrows():\n",
+    "        counter += 1\n",
+    "        print(f'{v.title} (score = {v.score})')\n",
+    "    \n",
+    "    print('\\n')\n",
+    "\n",
+    "    return df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "id": "67b3584d",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Most similar results to modern art in Europe in \"title\" namespace:\n",
+      "\n",
+      "Museum of Modern Art (score = 0.875286043)\n",
+      "Western Europe (score = 0.867383599)\n",
+      "Renaissance art (score = 0.864250064)\n",
+      "Pop art (score = 0.860506058)\n",
+      "Northern Europe (score = 0.854678154)\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "query_output = query_article('modern art in Europe','title')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "3e7ac79b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "Most similar results to Famous battles in Scottish history in \"content\" namespace:\n",
+      "\n",
+      "Battle of Bannockburn (score = 0.869324744)\n",
+      "Wars of Scottish Independence (score = 0.861479)\n",
+      "1651 (score = 0.852555931)\n",
+      "First War of Scottish Independence (score = 0.84969604)\n",
+      "Robert I of Scotland (score = 0.846192539)\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "content_query_output = query_article(\"Famous battles in Scottish history\",'content')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d939342f",
+   "metadata": {},
+   "source": [
+    "## Weaviate\n",
+    "\n",
+    "The other vector database option we'll explore here is **Weaviate**, which offers both a managed, SaaS option like Pinecone, as well as a self-hosted option. As we've already looked at a cloud vector database, we'll try the self-hosted option here.\n",
```

Comment:
One thing I noticed is that some of the notebook is in future tense (e.g., "we will do X"), some is in present tense (e.g., "we set up Y"), and some is in past tense ("we used Docker"). Worth doing another pass to make them all consistent.

Reply:
Updated to align with future tense